functional-tests/common/com.liferay.ide.functional.liferay/src/com/liferay/ide/functional/liferay/action/DialogAction.java

```diff
@@ -247,6 +251,37 @@ public void checkAutomaticSync() {
 
 	}
 
+	public class KaleoWorkflowValidationAction {
+
+		public String[] comboxItems() {
+			return getComboxItem().items();
+		}
+
+		public ComboBox getComboxItem() {
```

Comment:
I know this is comboBox for "Error", "Warning", "Ignore", we should rename this to be more clear.